src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -953,11 +957,15 @@ function updateNumScalePossibleValues(questionNumber) {
         $("#maxScaleBox"+idSuffix).val(max);
     }
     
-    if (step < 0.001) {
+    step = roundToThreeDp(step);
+    if (step == 0) {
         step = 0.001;
-        $("#stepBox"+idSuffix).val(step);
     }
-    
+    $("#stepBox" + idSuffix).val(step);        
+    if (isNaN(step)) {
+        $("#stepBox" + idSuffix).val("");        
```

Comment:
Ah okay. The code looks good, just need to make sure the tests will fail without the fix, as mentioned by @damithc